Src/LinearSolvers/MLMG/AMReX_MLMG.H

```diff
@@ -12,6 +12,13 @@ class MLMGT
 {
 public:
 
+    class error
+        : public std::runtime_error
+    {
+    public :
+        error(const std::string& what) : std::runtime_error(what) {}
```

Comment:
```suggestion
        using std::runtime_error::runtime_error;
```
This make MLMG::error inherit all constructors of std::runtime_error.